quality/integration/src/test/resources/cleanup_tables.sh

```diff
@@ -2,8 +2,8 @@
 set -o errexit
 set -o nounset
 
-#List of prefixes used in DTQ emo integration test
-table_prefixes=("gatekeeper_")
+#Table prefix for searching tables and table removal execution
+table_prefix=("gatekeeper_")
```

Comment:
what's the rationale behind this change?
variable type is still an array, but naming is not accurate

Reply:
yes, I meant to do `table_prefix="gatekeeper_"`